index.php

```diff
@@ -7,6 +7,35 @@
 // Creates a variable which points to the correct database and gives username and password
 $db = new PDO('mysql:host=db;dbname=dinosaurs;', 'root', 'password');
 
+/**
+ * Checks if the form has been submitted and if so returns the dinos from the submitted search. If not, returns all dinos
+ *
+ * @param PDO $db database
+ * @return string of dinos to be displayed
+ */
+function checkIfSearched(PDO $db): string {
+    if (isset($_GET['submit'])) {
+        return Museum::displayAllDinos(DinosaurHydrator::getSearchedDinos($db, $_GET['search'], $_GET['filter']));
+    } else {
+        return Museum::displayAllDinos(DinosaurHydrator::getAllDinos($db));
+    }
+}
+
+/**
+ * Checks if the form has been submitted and if so returns the value the user searched for, otherwise returns search
+ *
+ * @return string
+ */
+function searchedValue(): string {
+    if (isset($_GET['submit'])) {
+        return $_GET['search'];
```

Comment:
still not checking if $_GET['search'] exists although it may not be necessary atm it's an extra layer to prevent bugs.

Reply:
If you made this comment previously and I resolved it please check the resolved first before commenting again as I gave info as to why I resolved it without changing - in this case I asked Ash and it wasn't necessary